src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -521,14 +528,15 @@ private void AddFunctionDescriptors()
         /// <summary>
         /// Clean up any old files or directories.
         /// </summary>
-        private void CleanupFileSystem()
+        private void DelayedCleanupFileSystem()
         {
-            if (ScriptOptions.FileLoggingMode != FileLoggingMode.Never)
+            Utility.ExecuteAfterDelay(() =>
```

Comment:
we were already doing this in the background to avoid startup impact as the comment below suggested. Are you saying we have proof this was still a problem and that this change improves startup? By how much?

Reply:
I can discuss more offline, (#5134), but we want to remove any blocking calls to Azure Files outside of wwwroot during cold start. This is in preparation of performance improvements.